db/schema.rb

```diff
@@ -0,0 +1,56 @@
+# This file is auto-generated from the current state of the database. Instead
```

Comment:
Please add how to create the database and run the migrations in the `Getting Started` section of the README file. The screenshot is broken. Kindly review the README and make sure it's professional and complete as it's the first point of reference for a user of the project.